src/GitHub.InlineReviews/Services/PullRequestSessionService.cs

```diff
@@ -125,6 +134,80 @@ class PullRequestSessionService : IPullRequestSessionService
             throw new FileNotFoundException($"Couldn't find merge base between {baseSha} and {headSha}.");
         }
 
+        /// <inheritdoc/>
+        public async Task<IPullRequestReviewCommentModel> PostReviewComment(
```

Comment:
More new methods. This this PR depend on some other PR that hasn't been merged yet?